os/drivers/input/Kconfig

```diff
@@ -0,0 +1,27 @@
+menu "Touch Driver Support"
+
+menuconfig TOUCH
+        bool "LCD TOUCH Driver Support"
+        default n
+        ---help---
+                Drivers for LCD Touch Devices. These drivers support 
+		interfaces as defined in include/tinyara/input/touchscreen.h
+
+                This selection is necessary to enable support for LCD touch drivers in
+                drivers/input
+
+if TOUCH
+
+config TOUCH_IST415
+	bool
```

Comment:
If there is no explanation with `" "`, user can't enable this. How do you enable this? Could you confirm that you used the menuconfig to modify defconfig?